lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -421,6 +425,14 @@ def add_destination_address(xml, address)
         end
       end
 
+      def add_3ds(xml, options)
+        return unless three_d_secure = options[:three_d_secure]
+
+        xml.tag!(:AuthenticationECIInd, three_d_secure[:eci]) if three_d_secure[:eci]
+        xml.tag!(:CAVV, three_d_secure[:cavv]) if three_d_secure[:cavv]
```

Comment:
What about AAV? It's in the schema, not sure why they make a difference between the two but it seems that they are expecting this when the card is a mastercard.